tests/integrationtests/custom_cost_aggregation_testfiles/by_cpu.txt

```diff
@@ -0,0 +1,143 @@
+
+	CPU 0
+		
```

Comment:
what about these empty lines, where do they come from? is that kernel stuff that is missing a proper kallsyms? can you add that too please?